src/Event/ContaoCoreEvents.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Event;
+
+/**
+ * Defines Constants for all Contao events.
+ *
+ * @author Leo Feyer <https://github.com/leofeyer>
+ * @author Andreas Schempp <https://github.com/aschempp>
+ */
+final class ContaoCoreEvents
+{
+    /**
+     * The contao.image_sizes.all event is triggered when all image sizes are generated.
+     *
+     * @var string
+     *
+     * @see Contao\CoreBundle\Event\GetImageSizesEvent
+     */
+    const IMAGE_SIZES_ALL = 'contao.image_sizes.all';
```

Comment:
True, because that's the constant's name. I guess we can live with the missing "options" in the name?